android/app/src/main/java/app/organicmaps/MwmActivity.java

```diff
@@ -2301,7 +2301,7 @@ private void toggleTrackRecordingPP()
         && mPlacePageViewModel.getMapObject().getValue().isTrackRecording())
       mPlacePageViewModel.setMapObject(null);
     else
-      mPlacePageViewModel.setMapObject(new TrackRecording());
+      mPlacePageViewModel.setMapObject(new TrackRecording(this));
```

Comment:
So, what was the point in adding strings to sdk resources if you can pass them here?
Move strings from sdk to app module and pass to constructor title string